### PR TITLE
Added previous photo of the day with buttons to navigate to previous and todays photos.

### PR DESCRIPTION
**Pull request by Abhi200206**
Added button by using which user can navigate to the previous and today's photo of the day. This add new feature to the existing main branch from feature branch.
## screenshot

Screenshot 2024-04-11 122133

### CODE REVIEW

File: components/User.tsx [hunk 1]
```diff
@@ -31,8 +34,33 @@ export default function User() {
                 <div className="my-4">
                     <p className="font-bold">Welcome to Nasa Photo viewer</p>
                 </div>
+                <div className="flex justify-between mx-2 py-2 my-2">
+                    <div onClick={async()=>{
+                        setLoading(true);
```
Comment (timeowilliams) [inline components/User.tsx:39]: Can we put the function inside of the onClick as a separate function?
```suggestion
                        setLoading(true);
```

File: components/User.tsx [hunk 2]
```diff
@@ -31,8 +34,33 @@ export default function User() {
...
+                        setDate(`Date:${t}`);
+                        setLoading(false);
+                    }} className="rounded p-1 bg-slate-200 cursor-pointer"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" className="w-6 h-6">
+                        <path stroke-linecap="round" stroke-linejoin="round" d="M10.5 19.5 3 12m0 0 7.5-7.5M3 12h18" />
+                    </svg>
+                    </div>
+                    <div><p className="font-black">{date}</p></div>
+                    <div onClick={async()=>{
+                        setLoading(true);
+                        let {imgurl,t}:any=await today();
```
Comment (timeowilliams) [inline components/User.tsx:51]: Likewise here

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+## Description
+
+<!-- Put the PR description here -->
+
+## Related NOTION tickets
+
```
Comment (timeowilliams) [inline pull_request_template.md:6]: Good change

**Review by timeowilliams — CHANGES_REQUESTED** [review]

**Comment by Abhi200206** [comment]
created a separate function for the function inside of the onClick Handler.

Screenshot 2024-04-11 235938

File: components/User.tsx (hunk 1 above)
Comment (Abhi200206) [reply components/User.tsx]: done

**Review by timeowilliams — APPROVED** [review]

**Pull request merged**